backend/src/v5/routes/teamspaces/projects/models/common/tickets.js

```diff
@@ -142,10 +153,84 @@ const updateTicket = (isFed) => async (req, res) => {
 	}
 };
 
+const getTicketComment = (isFed) => async (req, res, next) => {
+	const { params } = req;
+	const { teamspace, project, model, ticket, comment } = params;
+
+	try {
+		const getCommentById = isFed ? getFedTicketComment : getConTicketComment;
+		req.commentData = await getCommentById(teamspace, project, model, ticket, comment);
+		await next();
+	} catch (err) {
+		// istanbul ignore next
+		respond(req, res, err);
+	}
+};
+
+const getTicketComments = (isFed) => async (req, res, next) => {
```

Comment:
Let's call this `getAllTicketComments` or something a bit different (or call the other one `GetCommentById` or something

Having 2 function where one is just a plural of the other is very confusion 😆

Reply:
Tbh putting `Ticket` before Comment is a bit redundant too it's pretty obvious base on the context of the file. But not important to change